test/Unit/NameConverterMultibyteTraitTest.php

```diff
@@ -87,8 +88,8 @@ public function studlyCapsProvider()
                 "Inc1udingNumb3rs4Gigglez"
             ],
             [ #7 multipass ... sorry, multibyte
-                "including á ßüñƈħ ǒf Multibyte ©h4r4C73r5",
-                "IncludingÁßüñƈħǑfMultibyteH4r4C73r5"
+                "including á büñƈħ ǒf Multibyte ©h4r4C73r5",
```

Comment:
Why change the ß?

I thought we found a fix for that

Reply:
My misunderstanding, I got it in my head that we was fine that it converted that way. Will fix that.